src/vendor/studio.scss

```diff
@@ -1130,15 +1129,25 @@ $bi-app-right           : right;
 // edx-platform/cms/static/sass/_base.scss
 html {
   font-size: 62.5%;
-  height: 102%; // force scrollbar to prevent jump when scroll appears, cannot use overflow because it breaks drag
+  height: 100%; // force scrollbar to prevent jump when scroll appears, cannot use overflow because it breaks drag
```

Comment:
This is the default value I believe. Is there a reason to set it here? Also the comment no longer applies.

Reply:
You are right comment isn't actual. `height: 100%` will be included in the next our pr's  regarding the task - `refactoring of remove global font-size from the html tag`. Comment also will be removed.

In this case here it's necessary for the correct display of the footer hyperlink.

Reply:
Ah great, thank you.